runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -203,19 +201,24 @@ private Class getStreamType(UUID streamId) {
     // In charge of summoning Corfu maps back in this world
     private ExecutorService necromancer;
 
-    private final Map<UUID, StreamMetaData> streamsMetaData;
+    private final Hashtable<UUID, StreamMetaData> streamsMetaData;
+    private AtomicInteger numCompletedCheckpoints;
+    AtomicReference<TrimmedException> trimmedException;
+    long minSnapshotAddress = Address.NON_EXIST;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651460)